app/server.js

```diff
@@ -0,0 +1,38 @@
+
+import express from 'express';
+import fetch from 'isomorphic-fetch';
+import { fileURLToPath } from 'url';
+import { dirname, join } from 'path';
+import { DaprClient, HttpMethod } from "@dapr/dapr";
+
+const app = express();
+app.use(express.json());
+
+const port = process.env.APP_PORT ?? '3000';
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+const publicPath = join(__dirname, 'public');
+
+const daprHost = "127.0.0.1";
+const daprPort = "9001";
+
+const client = new DaprClient({ daprHost, daprPort });
+
+const serviceAppId = "server";
+const serviceMethod = "health";
+
+
+app.get('/health', async (_req, res) => {
+    const response = await client.invoker.invoke(serviceAppId, serviceMethod, HttpMethod.GET);
+    console.log(response);
+    res.json(response);
+});
```

Comment:
I set up this endpoint to see if app is alive. In the near future we should split it up on java server in two separate where on is just a pong and the other with more appropriate name generates ID, do business logic and return it. You don't need to do it right now if you don't want to, just sayin'.

Reply:
Same here. Wanted to be sure that the client-server in js communicates well with an actual java server.

Reply:
so leave it, I'll do it someday